tests/testthat/helper-valid_log.R

```diff
@@ -67,14 +67,40 @@ expect_valid_log <- local({
       stdout = TRUE,
       stderr = TRUE
     )
+
+    # "inst/test/inspect/logs/2025-03-24T10-39-36-05-00_simple-arithmetic_fQ9mYnqZFhtEuUenPpJgKL.json"
+    if (length(result) == 1) {
+      formatted_message <- cli::format_message(c(
+        "The generated log did not pass the pydantic model:",
+        glue::glue("{{.field {result[1]}}}")
+      ))
+    } else {
```

Comment:
To simplify this control flow, I am going to remove this `else` and then return early from the `if` condition.